src/components/navBar.tsx

```diff
@@ -1,20 +1,33 @@
 "use client";
 
-import { useState } from "react";
+import { useState, useEffect } from "react";
+import { signal, useSignal } from "@preact/signals-react";
 
 import Image from "next/image";
 import SiteButton from "./siteButton";
 import Link from "next/link";
 import ShareOptions from "./shareOptions";
 import SignupOptions from "./signupOptions";
 
+export const dropDown = signal("mt-8");
+
 export default function NavBar() {
   const [clickedButton, setClickedButton] = useState("");
 
   function handleNavButtonClick(e: any) {
     setClickedButton(clickedButton === e.target.value ? "" : e.target.value);
   }
 
+  useEffect(() => {
```

Comment:
Why are you using signals here? It seems like you're just console.logging the value?

Reply:
I'm using a signal in the NavBar so I can send it's value over to the Donation Box - that way I can have it scootch out of the dropdown options button's way. I've updated the terms though, since I was using general ones while I was trying to figure it out 😆